eng/Version.Details.xml

```diff
@@ -33,6 +33,16 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>30ab651fcb4354552bd4891619a0bdd81e0ebdbf</Sha>
     </Dependency>
+    <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.0.0">
+      <Uri>https://github.com/dotnet/core-setup</Uri>
+      <Sha>7d57652f33493fa022125b7f63aad0d70c52d810</Sha>
+    </Dependency>
+    <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
+    <Dependency Name="Microsoft.Extensions.FileSystemGlobbing" Version="2.0.0">
+      <Uri>https://github.com/dotnet/core-setup</Uri>
+      <Sha>7d57652f33493fa022125b7f63aad0d70c52d810</Sha>
+    </Dependency>
```

Comment:
Wow, those versions are quite old. Aren't we afraid that full source build could break when using the live versions (8.0.0)?  Would it make sense to minimize the version diff?

Reply:
Source-build was using live versions until we switched to dependencies-only flow, few months ago: https://github.com/dotnet/installer/pull/17011 By adding these dependencies, we are essentially going back to live versions.

Granted, this big version difference is worrying, in the long-term.